extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -96,7 +96,7 @@ export class AmpAdUIHandler {
     if (this.containerElement_) {
       // Collapse the container element if there's one
       attemptCollapsePromise = this.element_
-        .getResources()
+        .getMutator()
```

Comment:
Can we just always call `Services.mutatorForDoc()` instead? We're currently trying to remove most of service getters from `BaseElement`.